charts/geoserver/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: geoserver
 description: Helm chart for the geoserver
 icon: https://geoserver.org/img/uploads/geoserver_icon.png
-version: 0.0.3
-appVersion: 2.21.2
+version: 0.0.4
```

Comment:
Would it make sense to use v1.0.0 here?

Reply:
v1.0.0 would mean, in short, that our API is stable and future minor or patch releases won't contain breaking changes. So I wouldn't make the jump to 1.0.0 because of this change. I don't think security updates necessarily have to be major releases.

But since the development in this repository is mostly finished we probably already should have released version 1.0.0 and could make up for that now. what do you think?

Reply:
I agree that this change in GeoServer helm chart isn't a breaking change of the helm chart itself.

However, IMHO, the current versions of all helm charts in this repository are worth to get released as v1.0.0. This could be done in a subsequent MR, right?

Reply:
yes, that makes sense

mapproxy has already version 1.1.0, I'll update the other helm charts in a follow-up